packages/stark-ui/src/modules/table/components/_table.component.scss

```diff
@@ -1,26 +1,21 @@
-//TODO: MatIcon size
-/*@mixin md-icon-size($size: 24px) {
-  font-size: $size;
-  height: $size;
-  width: $size;
-}
-
-.mat-icon{
-  @include md-icon-size(16px);
-  font-weight: inherit;
-  margin-left: 5px;
-}*/
-
 .stark-table {
   .header {
     align-items: center;
     display: flex;
     justify-content: space-between;
+    margin-bottom: 20px;
+
+    .transcluded{
```

Comment:
Could you please add a space before the bracket ?
(Prettier should do it automatically normally 😕)